backend/custom_models/migrations/0007_upload_emib_sample_test.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 2.1.7 on 2019-06-03 15:27
+
```

Comment:
Can you add a comment saying you made this - co it's clear its not one of the autogenerated ones.

Reply:
fixed in 137f4ab